content/yaml/testing.md

```diff
@@ -50,5 +50,36 @@ scripts:
 
 ## Flutter drive test
```

Comment:
Drive tests or Driver tests?
I'd also add an introductory sentence saying that Flutter driver tests can be run on emulator or simulator when building for Android or iOS or in a web browser when building for the web.

Reply:
Fixed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/146/commits/0c31b30a4a008e8d6f708454a8033844adfcb3a3